mmyolo/models/task_modules/assigners/batch_task_aligned_assigner.py

```diff
@@ -1,157 +1,285 @@
+        """Compute alignment metric between all bbox and gt.
 
-        pd_scores = pd_scores.permute(0, 2, 1)
+        Args:
+            gt_labels (Tensor): Ground true labels,
+                shape(batch_size, num_gt, 1)
+            gt_bboxes (Tensor): Ground true bboxes,
+                shape(batch_size, num_gt, 4)
+            pred_scores (Tensor): Scores of predict bbox,
+                shape(batch_size, num_priors, num_classes)
+            pred_bboxes (Tensor): Predict bboxes,
+                shape(batch_size, num_priors, 4)
+            batch_size (int): Batch size.
+            num_gt (int): Number of ground truth.
+        Returns:
+            alignment_metrics (Tensor): Align metric,
+                shape(batch_size, num_gt, num_priors)
+            overlaps (Tensor): Overlaps, shape(batch_size, num_gt, num_priors)
+        """
+        pred_scores = pred_scores.permute(0, 2, 1)
         gt_labels = gt_labels.to(torch.long)
-        ind = torch.zeros([2, self.bs, self.n_max_boxes], dtype=torch.long)
-        ind[0] = torch.arange(end=self.bs).view(-1, 1).repeat(1, self.n_max_boxes)
+        ind = torch.zeros([2, batch_size, num_gt], dtype=torch.long)
+        ind[0] = torch.arange(end=batch_size).view(-1, 1).repeat(1, num_gt)
         ind[1] = gt_labels.squeeze(-1)
-        bbox_scores = pd_scores[ind[0], ind[1]]
+        bbox_scores = pred_scores[ind[0], ind[1]]
 
-        overlaps = self.iou_calculator(gt_bboxes, pd_bboxes)
-        align_metric = bbox_scores.pow(self.alpha) * overlaps.pow(self.beta)
+        overlaps = self.iou_calculator(gt_bboxes, pred_bboxes)
+        alignment_metrics = bbox_scores.pow(self.alpha) * overlaps.pow(
+            self.beta)
 
-        return align_metric, overlaps
+        return alignment_metrics, overlaps
 
     def select_topk_candidates(self,
-                               metrics,  
-                               largest=True, 
-                               topk_mask=None):
+                               align_scores: Tensor,
+                               using_largest_topk: bool = True,
+                               topk_mask: Optional[Tensor] = None) -> Tensor:
+        """Compute alignment metric between all bbox and gt.
 
-        num_anchors = metrics.shape[-1]
+        Args:
+            align_scores (Tensor): Alignment scores of predict bboxes,
+                shape(batch_size, num_gt, num_priors)
+            using_largest_topk (bool): Controls whether to using largest or
+                smallest elements.
+            topk_mask (Tensor): Topk mask, shape(batch_size, num_gt, self.topk)
+        Returns:
+            Tensor: Topk candidates mask, shape(batch_size, num_gt, num_priors)
+        """
+        num_priors = align_scores.shape[-1]
         topk_metrics, topk_idxs = torch.topk(
-            metrics, self.topk, axis=-1, largest=largest)
+            align_scores, self.topk, axis=-1, largest=using_largest_topk)
```

Comment:
Which is better: align_metric or align_score? If it is determined then all code naming should be consistent